### DISCUSSION TITLE
Texture on the iCub Simulator

### DISCUSSION THREAD

**Discussion by LakshithaPW** [robotology/community#217] [Q&A] [answered]
Hello,

I have a query regarding the texture on the ICub simulator  (Specifically the Bluetexture on the cover).
When blue texture is applied, it will be applied on the robot inside-out which is a strange thing.
I was wondering whether there is some solution without reinstalling the iCub Simulator.
Please let me know if you have some idea about the reason for this.

Thanks
lakshitha

**Comment by vtikha** [comment, marked as answer]
@LakshithaPW No, the blue texture is also applied on the inside of the cover. Maybe I did not fully understand but I do not see it as a strange thing eg: this also is the same thing on the real robot.

**Comment by LakshithaPW** [comment]
@vtikha Thanks a lot, I tried to reinstall the simulator and it seems to be working correctly now.

**Comment by jimingda** [comment]
Hi, 

are there texture file with stl about icub cover in gazebo, or if so, where can i find them?

Thanks.
